core/test/utilities/helpers/escaping-helpers.spec.js

```diff
@@ -16,9 +24,90 @@ describe('Iframe-helpers', () => {
     assert.equal(sanitizedParam, '&lt;&gt;&quot;&#39;&sol;');
   });
 
-  it('escapeRegExp', () => {
-    const regex = '/*/';
-    const escapedRegexp = EscapingHelpers.escapeRegExp(regex);
-    assert.equal(escapedRegexp, '\\/\\*\\/');
+  it('escapeKeyForRegexp', () => {
+    const key = 'some/*/thing';
+    const escapedRegexp = EscapingHelpers.escapeKeyForRegexp(key);
+    assert.equal(escapedRegexp, 'some\\/\\*\\/thing');
+  });
+
+  describe('processTextAndLinks', () => {
+    it('without links', () => {
```

Comment:
please also create stubs for external function in this testcase, maybe you can add them to a beforeEach block :)

Reply:
And maybe it is a good idea to have an additional testcase to check that the stub functions are called with proper values

Reply:
done